README.md

```diff
@@ -1,6 +1,6 @@
 # HCB: what does it stand for? 
 
-First, a bit of context from 31/08/2023:
+First, a bit of context from 2023-08-31:
```

Comment:
```suggestion
First, a bit of context:
```

I think we can just remove this. First off, we have version control, and if someone really wants more info, they can click on the link.